proto/storage/collector_runtime_config.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
+
+package storage;
+
+import "storage/resource_collection.proto";
+
+option go_package = "storage";
+option java_package = "io.stackrox.proto.storage";
+
+message CollectorRuntimeConfigACK {
+  string error = 1;
+  Action action = 2;
+
+  enum Action {
+    ACK = 0;
+    NACK = 1;
+  }
+}
+
+enum RuntimeFeature {
+  PROCESSES = 0;
+  NETWORK_CONNECTIONS = 1;
+  NETWORK_ENDPOINTS = 2;
+}
+
+message Status {
+  message ProcessStatus {
+    bool enabled = 1;
+  }
+
+  message NetworkConnectionStatus {
+    bool enabled = 1;
+    bool aggregate_external = 2;
+  }
+
+  message NetworkEndpointStatus {
+    bool enabled = 2;
+    bool processes_listening_on_port = 3;
```

Comment:
Numbering should start at 1

Reply:
Done

Reply:
Resolving as the comment was addressed and @ovalenti is on PTO.